components/account/AccountHeader.vue

```diff
@@ -52,7 +52,7 @@ function previewAvatar() {
 }
 
 async function toggleNotifications() {
-  const dataGlean = relationship?.notifying ? 'profile.notify_stop' : 'profile.notify_start'
+  const dataGlean = relationship?.notifying ? 'profile.notify.stop' : 'profile.notify.start'
```

Comment:
These events were renamed to include the `.` separation between `notify` and the action (`start` or `stop`)